src/main/java/com/algorand/algosdk/builder/transaction/TransactionBuilder.java

```diff
@@ -90,7 +90,7 @@ final public Transaction build() {
 
     /**
      * Query the V1 REST API with {@link AlgodApi} for Transaction Parameters:
-     * Initialize fee, genesisID, genesisHash, firstValid, lastValid using {@link TransactionParams}.
+     * Initialize fee, genesisID, genesisHash, firstValid, lastValid if not already set using {@link TransactionParams}.
```

Comment:
```suggestion
     * Initialize fee, genesisID, genesisHash, firstValid, lastValid using {@link TransactionParams} if not already set.
```
This makes more sense to me.